cmd/git-sync/main.go

```diff
@@ -55,6 +55,8 @@ var flRev = flag.String("rev", envString("GIT_SYNC_REV", "HEAD"),
 	"the git revision (tag or hash) to check out")
 var flDepth = flag.Int("depth", envInt("GIT_SYNC_DEPTH", 0),
 	"use a shallow clone with a history truncated to the specified number of commits")
+var flSubmoduleMode = flag.String("submodule-mode", envString("GIT_SYNC_SUBMODULE_MODE", "recursive"),
```

Comment:
I am not a big fan of "mode" as a term for things like flags.  I don't think it adds much.  Is `--submodules` much worse in your opinion?

Reply:
it's not, yeah "mode" is weird here... i always think working with enum like things is weird with things like command like flags